### ISSUE TITLE
StarLite Mk V battery always at 0%, charging LED red, as soon as power is disconnected, it warns about auto-shutoff

### ISSUE THREAD

**Issue by dchang0** [StarLabsLtd/firmware#211]
This is the flip side of the other issue I had where the battery level was always at 100%.

This one is more reproducible, though.
If I take my tablet, which is turned completely off and has been for a while--and I know the battery charge is not full but also not empty (usually it's around 50 to 60%)-- and I plug in the charger and then turn on the power, the battery indicator will claim it is 0% forever, even while the charging LED is red.

If I unplug the charging cable while the tablet is on, it will immediately warn that it will automatically shut down due to low battery. I can then turn it off, wait 12 seconds, turn it on, then see the charge level at about what it used to be (50 to 60%). I can then plug in the charger and see normal charging behavior. The charging that was supposedly happening (when the charging LED was red) does not actually increase the battery's level, even if left there a long time.

Basically, the rule I have to follow is never plug in the charger while the tablet is off; I have to turn the tablet on first, then I can actually charge and see the battery level increase. I usually charge in suspend.

BIOS is 24.07.

**Comment by Sean-StarLabs** [comment]
Please try on 24.08. If that doesn't work, ping support
